src/main/java/frc/robot/Constants.java

```diff
@@ -18,14 +19,20 @@ public static class MotorIDConstants {
     public static final int kElevatorMotor1 = 1;
     public static final int kElevatorMotor2 = 2;
     public static final int kIntakeMotor = 9;
+    public static final int kPivotMotor = 0; // FIXME: Change to correct ID
+    public static final int kConveyorMotor = 0; // FIXME: Change to correct ID
     // CANavor Can Bus
   }
 
   // Intake Constants
   public static class IntakeConstants {
-    public static final double kSpeed = 0.5;
-  }
+    public static final double kIntakeSpeed = 0.5;
+    public static final double kPivotSpeed = 0.5;
```

Comment:
You won't need kPivotSpeed.  You will want to use closed loop control for the pivot.  

intake and conveyer can be usually run open loop.  ( Where speed is a percentage of the maximum possible speed of the motors. )

You will need constants for PID.  Look in saddlebags for a PID class to reuse

Reply:
We will probably want to start by testing it with just raw power, and not using PID, just to make sure that we have our setpoints right.
Also the HowdyPID class is not worth using at the moment because It's not updated to the new REVlib, and will only work with the RIO's PID controller, so I would recomend just using the CTRE PID stuff